libtiledbvcf/src/read/reader.cc

```diff
@@ -45,6 +45,67 @@
 namespace tiledb {
 namespace vcf {
 
+AlleleCountKey::AlleleCountKey(
```

Comment:
Did you want this implementation of `AlleleCountKey` to be in `allele_count.cc` to match the location of the `AlleleCountKey` definition in `allele_count.h`?